cli/tsc/dts/lib.deno.ns.d.ts

```diff
@@ -6688,4 +6688,433 @@ declare namespace Deno {
         | (ServeTlsOptions & TlsCertifiedKeyOptions)
       ),
   ): HttpServer<Deno.NetAddr>;
+
+  // To update, input the contents of `cli/schemas/config-file.v1.json` into
+  // https://transform.tools/json-schema-to-typescript
+  /**
+   * Interface for the
+   * {@link https://docs.deno.com/runtime/manual/getting_started/configuration_file | Deno configuration file}.
+   *
+   * Note: It is recommended to only use this interface when dynamically
+   * importing a JSON file that is not analyzable, as analyzable imports of JSON
+   * files provide file-based type-hinting and autocompletion anyway.
```

Comment:
I should be fact-checked on this, namely if the wording is accurate. These are based on what I found on my machine.